src/main/menu.main.ts

```diff
@@ -170,6 +172,12 @@ export class MenuMain extends BaseMenu {
                 click: () => this.main.messagingService.send('copyPassword'),
                 accelerator: 'CmdOrCtrl+P',
             },
+            {
+                label: this.main.i18nService.t('copyVerificationCodeTotp'),
+                id: 'copyTotp',
+                click: () => this.main.messagingService.send('copyTotp'),
+                accelerator: 'CmdOrCtrl+T',
+            }
```

Comment:
To this line as well, after the trailing `}` 😁  (see above `{ ... }` object block.

Reply:
you're too fast :D